.github/workflows/tests.yml

```diff
@@ -189,10 +189,13 @@ jobs:
         include:
           - name: tests (no-compatibility mode)
             test-flags: --no-compat --online --coverage
+            buildpulse: false
           - name: tests (generic OS)
             test-flags: --generic --online --coverage
+            buildpulse: true
```

Comment:
Might want to pick the macOS job for this because it runs the most tests (e.g. `brew cask` ones)

Reply:
Cool. That's done. 

Note that it includes [this hack](https://github.com/jasonrudolph/brew/pull/2/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR272-R275) just for the purposes of getting things to run within this fork. That hack won't be part of the upstream PR. 😇